README.markdown

```diff
@@ -47,6 +47,9 @@ You can specify `headers`, `params`, and `body` when calling a `CRUD` method. Fo
 
 ```ruby
 gibbon.lists.retrieve(headers: {"SomeHeader": "SomeHeaderValue"}, params: {"query_param": "query_param_value"})
+gibbon.lists.create(headers: {"SomeHeader": "SomeHeaderValue"}, params: {"query_param": "query_param_value"}, body: {"SomeBody": "SomeBodyValue"})
+gibbon.lists(list_id).update(headers: {"SomeHeader": "SomeHeaderValue"}, params: {"query_param": "query_param_value"}, body: {"SomeBody": "SomeBodyValue"})
+gibbon.lists(list_id).delete(headers: {"SomeHeader": "SomeHeaderValue"}, params: {"query_param": "query_param_value"})
```

Comment:
You've left out `upsert` here. I don't see tremendous value in adding examples since there are examples of all of these except delete elsewhere in the readme, but if we're going do add them then let's be complete.